### PR TITLE
Minor text changes

### PR DESCRIPTION
**Pull request by hyanwong**
Just doing a pass through the first section, and I made a few minor changes. I thought I should PR these before moving to the next section.

Depends on the resolution to #206

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -139,19 +142,21 @@ \section*{Ancestral graphs and stochastic processes}
...
-The Ancestral Recombination Graph (ARG) was later introduced by
-Griffiths~\citep{griffiths1991two,ethier1990two,griffiths1996ancestral,
-griffiths1997ancestral}, and is a
-closely related and complementary treatment of the same process. Where
-Hudson's algorithm is focused on the efficient \emph{simulation} of the
-process, the original ARG literature is focused on mathematical
-results about the stochastic process. Griffiths and colleagues used the word
+Although Hudson's algorithm was accompanied by a graph
+schematic~\citep[][Fig. 5]{hudson1983properties}, the term
+"Ancestral Recombination Graph" or ARG, was introduced later by
```
Comment (jeromekelleher) [inline paper.tex:150]:
```suggestion
``Ancestral Recombination Graph'' or ARG, was introduced later by
```

**Review by jeromekelleher — APPROVED** [review]
LGTM - we need a CI check for TeX quotes!

File: paper.tex (hunk 1 above)
Comment (hyanwong) [reply paper.tex:150]: Damn, I thought I had caught all those! Thanks for spotting.

Comment (jeromekelleher) [reply paper.tex]: The GitHub diff shows them up very clearly

**Pull request merged**